lib/heist_definitions.dart

```diff
@@ -1,30 +1,55 @@
-part of heist;
+import 'package:flutter/foundation.dart';
+
+const String Succeed = 'SUCCEED';
```

Comment:
I'm too used to constants in capitals. I guess it's part of the language :(

Reply:
I don't mind but android studio complains about not using the dart conventions...